---
layout: publications
title: Publications
permalink: /publications/
---

- R. Vicedomini, F. Andreace, Y. Dufresne, R. Chikhi, C. Duitama González,
**MUSET: Set of utilities for constructing abundance unitig matrices from sequencing data**,
*Bioinformatics* (2025) doi:&nbsp;[10.1093/bioinformatics/btaf054](https://doi.org/10.1093/bioinformatics/btaf054)

- S. Liu *et al.*,
**Analysis of metagenomic data**,
*Nature Reviews Methods Primers* (2025) doi:&nbsp;[10.1038/s43586-024-00376-6](https://doi.org/10.1038/s43586-024-00376-6)

- C. Duitama Gonzalez, R. Vicedomini, T. Lemane, N. Rascovan, H. Richard, R. Chikhi,
**decOM: similarity-based microbial source tracking of ancient oral samples using k-mer-based methods**,
*Microbiome* (2023) doi:&nbsp;[10.1186/s40168-023-01670-3](https://doi.org/10.1186/s40168-023-01670-3)

- C. Duitama Gonzalez, S. Rangavittal, R. Vicedomini, R. Chikhi, H. Richard,
**aKmerBroom: Ancient oral DNA decontamination using Bloom filters on k-mer sets**,
*Iscience* (2023). doi:&nbsp;[10.1016/j.isci.2023.108057](https://doi.org/10.1016/j.isci.2023.108057)

- F. Meyer *et al.*,
**Critical Assessment of Metagenome Interpretation: the second round of challenges**,
*Nature Methods* (2022). doi:&nbsp;[10.1038/s41592-022-01431-4](https://doi.org/10.1038/s41592-022-01431-4)

- R. Vicedomini\*, J.-P. Bouly\*, E. Laine, A. Falciatore &amp; A. Carbone,
**Multiple Profile Models Extract Features from Protein Sequence Data and Resolve Functional Diversity of Very Different Protein Families**,
*Molecular Biology and Evolution* (2022). doi:&nbsp;[10.1093/molbev/msac070](https://doi.org/10.1093/molbev/msac070)

- R. Vicedomini, C. Quince, A. E. Darling &amp; R. Chikhi,
**Strainberry: automated strain separation in low-complexity metagenomes using long reads**,
*Nature Communications* (2021). doi:&nbsp;[10.1038/s41467-021-24515-9](https://doi.org/10.1038/s41467-021-24515-9)

- D. Danko *et al.*,
**A global metagenomic map of urban microbiomes and antimicrobial resistance**,
*Cell* (2021). doi:&nbsp;[10.1016/j.cell.2021.05.002](https://doi.org/10.1016/j.cell.2021.05.002)

- R. Vicedomini, C. Blachon, F. Oteri &amp; A. Carbone,
**MyCLADE: a multi-source domain annotation server for sequence functional exploration**,
*Nucleic Acids Research*, Web Server Issue (2021). doi:&nbsp;[10.1093/nar/gkab395](https://doi.org/10.1093/nar/gkab395)

- L. David, R. Vicedomini, H. Richard &amp; A. Carbone,
**Targeted domain assembly for fast functional profiling of metagenomic datasets with S3A**,
*Bioinformatics* (2020). doi:&nbsp;[10.1093/bioinformatics/btaa272](https://doi.org/10.1093/bioinformatics/btaa272)

- A. Ugarte\*, R. Vicedomini\*, J. S. Bernardes &amp; A. Carbone,
**A multi-source domain annotation pipeline for quantitative metagenomic and metatranscriptomic functional profiling**,
*Microbiome* (2018). doi:&nbsp;[10.1186/s40168-018-0532-2](https://doi.org/10.1186/s40168-018-0532-2)

- R. Vicedomini, F. Vezzi, S. Scalabrin, L. Arvestad &amp; A. Policriti,
**Hierarchical assembly of pools**,
*Bioinformatics and Biomedical Engineering, Lecture Notes in Computer Science, Springer International Publishing* (2015). doi:&nbsp;[10.1007/978-3-319-16480-9_21](https://doi.org/10.1007/978-3-319-16480-9_21)

- K. R. Bradnam *et al.*,
**Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species**,
*Gigascience* (2013). doi:&nbsp;[10.1186/2047-217X-2-10](https://doi.org/10.1186/2047-217X-2-10)

- B. Nystedt, N. R. Street *et al.*,
**The Norway spruce genome sequence and conifer genome evolution**,
*Nature* (2013). doi:&nbsp;[10.1038/nature12211](https://doi.org/10.1038/nature12211)

- R. Vicedomini, F. Vezzi, S. Scalabrin, L. Arvestad &amp; A. Policriti,
**GAM-NGS: genomic assemblies merger for next generation sequencing**
*BMC Bioinformatics* (2013). doi:&nbsp;[10.1186/1471-2105-14-S7-S6](https://doi.org/10.1186/1471-2105-14-S7-S6)

# Pre-prints

- R. Vicedomini\*, N. Righetti\*, L. Polit\*, S. Condemi, L. Longo &amp; A. Carbone,
**Genetic dietary adaptation in Neandertal, Denisovan and Sapiens revealed by gene copy number variation**
*bioRxiv* (2024). doi:&nbsp;[10.1101/2021.10.30.466563](https://doi.org/10.1101/2021.10.30.466563)

# PhD Thesis

- Vicedomini R.,
**Alignment and reconciliation strategies for large-scale de novo assembly**,
*Doctoral Thesis, Università degli Studi di Udine* (2016). [[pdf](https://air.uniud.it/bitstream/11390/1132931/1/10990_684_thesis_final_pdfa.pdf)]
